Symfony/CS/Fixer/Symfony/PhpdocParamsFixer.php

```diff
@@ -56,22 +56,25 @@ public function fix(\SplFileInfo $file, $content)
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
+    public function getDescription()
     {
-        // should be run after the PhpdocIndentFixer
-        // this is because this fixer currently only deals with docblocks that
-        // are correctly indented, and skips those that are not
-        return -10;
+        return 'All items of the @param, @throws, @return, @var, and @type phpdoc tags must be aligned vertically.';
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getDescription()
+    public function getPriority()
     {
-        return 'All items of the @param, @throws, @return, @var, and @type phpdoc tags must be aligned vertically.';
+        // should be run after all other phpdoc fixers
+        return -10;
```

Comment:
Sorry but it is not obvious for me why, could you explain ?
Again eg type vs var...